### PR TITLE
Add information on SEntry and heroku logs

### PR DESCRIPTION
**Pull request by yarikoptic**

### CODE REVIEW

File: docs/40_development.md [hunk 1]
```diff
@@ -86,6 +86,35 @@ controls Netlify settings. The @dandibot GitHub account is the "owner" of the
...
+## Logging
+
+### Sentry
+
+Sentry is used for error tracking main deployment.
+To access Sentry, login to https://dandiarchive.sentry.io .
+
+### Heroku & Papertrail
+
+`dandi-api` and `dandi-api-staging` apps have Papertrail add-on configured to capture logs.
```
Comment (waxlamp) [inline docs/40_development.md:107]:
```suggestion
The `dandi-api` and `dandi-api-staging` apps have the Papertrail add-on configured to capture logs.
```

File: docs/40_development.md [hunk 2]
```diff
@@ -86,6 +86,35 @@ controls Netlify settings. The @dandibot GitHub account is the "owner" of the
...
+To access papertrail login to [Heroku dashboard](https://dashboard.heroku.com), proceed to the corresponding app and click on "Papertrail" add-on.
```
Comment (waxlamp) [inline docs/40_development.md:108]:
```suggestion
To access Papertrail, log in to the [Heroku dashboard](https://dashboard.heroku.com), proceed to the corresponding app and click on the "Papertrail" add-on.
```

File: docs/40_development.md [hunk 3]
```diff
@@ -86,6 +86,35 @@ controls Netlify settings. The @dandibot GitHub account is the "owner" of the
...
+
+CRON job on `drogon` server backups Papertrail logs as .csv files hourly at `/mnt/backup/dandi/papertrail-logs/{app}`.
```
Comment (waxlamp) [inline docs/40_development.md:110]:
```suggestion
A cronjob on the `drogon` server backs up Papertrail logs as .csv files hourly at `/mnt/backup/dandi/papertrail-logs/{app}`.
```

Comment (waxlamp) [reply docs/40_development.md:110]: Just out of curiosity, why bother to do this? Papertrail itself backs up monthly log bundles for the previous year.

File: docs/40_development.md [hunk 4]
```diff
@@ -86,6 +86,35 @@ controls Netlify settings. The @dandibot GitHub account is the "owner" of the
...
+Moreover, `heroku logs` processes per app dump logs to `/mnt/backup/dandi/heroku-logs/{app}` directory.
```
Comment (waxlamp) [inline docs/40_development.md:111]: Unclear what this refers to. Another process on `drogon`?

Comment (yarikoptic) [reply docs/40_development.md:111]: Right! Wording is indeed unclear although sounded fine for me originally ;-)  Is this making more sense?

```suggestion
Another CRON job on `drogon` runs `heroku logs` processes per app to dump logs to `/mnt/backup/dandi/heroku-logs/{app}` directory for each hour.
```

**Pull request merged**